build/verify-feature-completeness/pom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.tools.eclipse</groupId>
+    <artifactId>build</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>verify-feature-completeness</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Installation Testing for GCP</name>
+  <description>
+    Attempt to install the main features with an Eclipse installation
+    to ensure that our feature definitions are complete.
+  </description>
+
+  <properties>
+    <installIUs>org.eclipse.platform.feature.group,com.google.cloud.tools.eclipse.suite.e45.feature.feature.group</installIUs>
+    <build.rootUri>${project.baseUri}/../..</build.rootUri>
+    <eclipse.repoUrl>${build.rootUri}/eclipse/ide-target-platform/target/repository</eclipse.repoUrl>
+    <gcp.repoUrl>${build.rootUri}/gcp-repo/target/repository</gcp.repoUrl>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud.tools.eclipse</groupId>
+      <artifactId>ide-target-platform.repo</artifactId>
+      <version>0.1.0-SNAPSHOT</version>
+      <type>eclipse-repository</type>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud.tools.eclipse</groupId>
+      <artifactId>google-cloud-plugin-eclipse.repo</artifactId>
+      <version>0.1.0-SNAPSHOT</version>
+      <type>eclipse-repository</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <!-- force a clean to remove any existing installation -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-clean-plugin</artifactId>
+        <version>2.5</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>clean</goal>
+            </goals>
+            <phase>package</phase>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.eclipse.tycho.extras</groupId>
+        <artifactId>tycho-eclipserun-plugin</artifactId>
+        <version>${tycho.version}</version>
+        <configuration>
+          <argLine>-Declipse.p2.mirrors=false</argLine>
+          <appArgLine>
+            -consoleLog -nosplash
+            -application org.eclipse.equinox.p2.director
+            -destination ${project.build.directory}/p2-installation
+            -repository ${eclipse.repoUrl},${gcp.repoUrl} 
+            -installIUs ${installIUs}
+          </appArgLine>
+          <repositories>
+            <repository>
+              <id>juno</id>
```

Comment:
Changed to just be 'eclipse' since it pulls whatever is in the current reactor, whether than be Mars or Neon.